config/settings.py

```
@@ -0,0 +1,143 @@
+    },
+]
+
+WSGI_APPLICATION = "config.wsgi.application"
+
+
+# Database
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
+    }
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)